### PR TITLE
Add default keymap icon hint

### LINKED ISSUE #167: Default keymap icon hint
It would be useful to provide some explanation for the user regarding the star icon.

My idea is that when hovering over the filled star icon of the side menu or the filled star icon next to the default keymap title we could display the following text:

> This is the default keymap which gets activated when powering the keyboard.

When hovering over the unfilled star icon of a non-default keymap the following text should be displayed:

> Makes this keymap the default keymap which gets activated when powering the keyboard.

### PR DESCRIPTION
**Pull request by fjozsef**
Fixes #167

### CODE REVIEW

**Comment by mondalaci** [comment]
Looks perfect.

File: src/store/reducers/keymap.ts [hunk 1]
```diff
@@ -62,7 +62,12 @@ export default function(state = initialState, action: Action): KeymapState {
 
         case KeymapActions.SET_DEFAULT:
             newState = state.entities.map((keymap: Keymap) => {
-                keymap.isDefault = (keymap.abbreviation === action.payload);
+                if (keymap.abbreviation === action.payload || keymap.isDefault) {
```
Comment (NejcZdovc) [inline src/store/reducers/keymap.ts]: Why did you add or operator with `keymap.isDefault`?

File: src/store/reducers/keymap.ts [hunk 2]
```diff
@@ -62,7 +62,12 @@ export default function(state = initialState, action: Action): KeymapState {
...
+                    let newKeymap: Keymap = new Keymap();
+                    Object.assign(newKeymap, keymap);
+                    keymap = newKeymap;
```
Comment (NejcZdovc) [inline src/store/reducers/keymap.ts]: Why are you setting newKeymap to the old one. Why don't just do something like:

```
Object.assign(newKeymap, keymap);
newKeymap.isDefault = keymap.abbreviation === action.payload;
return newKeymap
```

Comment (fjozsef) [inline src/store/reducers/keymap.ts]: I wanted to have one return statement., because this way the code more readable for me. 
If you need a new keymap, then you create a new and replace the old one, otherwise it remains the same. I know that your approach does the same, but I think mine is easier to understand.
